testing/src/main/java/org/oppia/testing/FakeEventLogger.kt

```diff
@@ -0,0 +1,32 @@
+package org.oppia.testing
+
+import android.content.Context
+import android.os.Bundle
+import org.oppia.testing.TestLogReportingModule.Companion.eventNameList
+import org.oppia.util.firebase.EventLogger
+
+/**
+ * A test specific fake event logger that adds the title of logged events to a list of strings.
+ */
+class FakeEventLogger : EventLogger {
+
+  /** This is used to add the title of the logged event to a list of titles. */
+  override fun logEvent(context: Context, bundle: Bundle, title: String) {
+    eventNameList.add(title)
+  }
+
+  /** This is used to get the most recent event title present in the list of strings. */
+  fun getMostRecentTitle(): String {
```

Comment:
I think you should return the entire event and not just the title because we would want to verify other parts of the event as well

Reply:
Done.